docs/contributor/06-20-environments-cleanup-cronjob.md

```diff
@@ -2,14 +2,17 @@
 
 >**CAUTION:** The CronJob must run only in the development environment. You must be connected to the development Kubernetes cluster before applying the CronJob.
 
+>**NOTE:** Environments Cleanup CronJob is applied manually, there is no automatic release process for the job because of its destructiveness. To get more details on how to apply the job visit its [`README`](../../utils/kyma-environments-cleanup-job/README.md). 
```

Comment:
```suggestion
>**NOTE:** Environments Cleanup CronJob is applied manually. There is no automatic release process for the job because of its destructiveness. To get more details on using the job, visit its [`README`](../../utils/kyma-environments-cleanup-job/README.md). 
```